pkgs/by-name/ni/nix-plugin-pijul/package.nix

```diff
@@ -11,6 +11,7 @@
   runCommand,
   pijul,
   nixVersions,
+  nixVersion ? "nix_2_18",
```

Comment:
How about `nixOverride ? null` and then `nix = if nixOverride == null then nixVersions.nix_2_18 else nixOverride`, instead of the strings. That way it is much more straight forward to override the Nix version again (e.g. with config.nix.package).

Reply:
Sounds good! Switched to `nixOverride`.

Reply:
Nothing outside of this package overrides its `nix`. Since we’re pinning one specific Nix, maybe we can just drop the internal `.override` from the test and continue referencing `nixVersions.nix_2_18` directly to avoid some complexity here?

Reply:
Oh, I’m sorry, I see that we test it against multiple Nix versions. However will that even work if it doesn’t build against 2.24?